ocrd_validator.py

```diff
@@ -205,33 +205,33 @@ def __validate_min_amount_of_tokens(self, line_num, line, min_amount):
         if len(line) < min_amount:
             expected = f"At least {min_amount} tokens."
             got = f"{len(line)} tokens"
-            hint = 'TOKEN_ERROR04: Each line must have a processor call, an input file, and an output file'
+            hint = 'LINE_ERROR_RULE_02: Each line must have a processor call, an input file, and an output file'
```

Comment:
The hint section should not include the error rule. Better to create an info variable and pass that variable to the print_syntax_error method. E.g., `info = "LINE_ERROR_RULE_02: Not enough amount of tokens"`

The hint should remain in this format:
`hint = "Each line must ..."`

And, btw, maybe we do not have to specify TOKEN/LINE in front of the "ERROR_RULE" to have a similar standard. This is just an idea, for now, you can keep it as you wish.